molecule/default/molecule.yml

```diff
@@ -0,0 +1,43 @@
+---
+dependency:
+  name: galaxy
+  options:
+    requirements-file: molecule/default/requirements.yml
+driver:
+  name: docker
+# lint: |
+#   set -e
+#   yamllint .
+#   flake8
+#   ansible-lint --force-color
+platforms:
+  - name: instance
+    image: docker.io/centos:8
```

Comment:
@hcherukuri we need to run it in privileged mode with systemd enabled, like we did for the [Wildfly collection](https://github.com/ansible-middleware/wildfly/blob/main/molecule/standalone/molecule.yml#L6).

Can you update this descriptor accordingly? Thanks!

Reply:
Updated.